public/app/features/logs/components/getLogRowStyles.ts

```diff
@@ -120,6 +120,11 @@ export const getLogRowStyles = memoizeOne((theme: GrafanaTheme2) => {
     logIconError: css`
       color: ${theme.colors.warning.main};
```

Comment:
Would you mind doing the same change (position relative, top: -2px, etc) to the `logIconError` please? Thanks!